### PR TITLE
Group dependabot updates - non-BBC

### PR DESCRIPTION
**Pull request by j-pendlebury**
**Overall change:** 
- https://github.com/bbc/simorgh/pull/5558
- https://github.com/bbc/simorgh/pull/5559
- @babel/preset-env@7.8.4
- amphtml-validator@1.0.30
- eslint-plugin-cypress@2.10.1
- depcheck@0.9.2
- eslint-plugin-import@2.20.1
- @storybook/addon-knobs@5.3.13
- eslint-plugin-react-hooks@2.4.0
- eslint-plugin-react@7.18.3
- @storybook/react@5.3.13
- husky@4.2.3
- storybook-chromatic@3.5.2
- stylelint-processor-styled-components@1.10.0
- styled-normalize@8.0.7
- lint-staged@10.0.7
- stylelint@13.2.0
- jsdom@16.2.0
- terser-webpack-plugin@2.3.5
- webpack-cli@3.3.11
- webpack@4.41.6

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] I have added labels to this PR for the relevant pod(s) affected by these changes
- [x] I have assigned this PR to the Simorgh project

**Testing:**
- ~~Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)~~
- ~~If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`)~~
- [x] This PR requires manual testing

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -156,51 +156,51 @@
     "chrome-launcher": "^0.12.0",
     "compression-webpack-plugin": "^3.1.0",
     "copy-webpack-plugin": "^5.1.1",
-    "cypress": "3.8.1",
-    "cypress-multi-reporters": "^1.2.3",
-    "depcheck": "^0.9.1",
+    "cypress": "4.0.2",
```
Comment (amywalkerdev) [inline package.json]: We downgraded the cypress version as a measure to see if it helped the test run time. I can't find the issue to bump it back up... Can we check and see if this impacts the running time of some of the e2es?

**Review by amywalkerdev — DISMISSED** [review]

File: package.json (hunk 1 above)
Comment (j-pendlebury) [reply package.json]: > We downgraded the cypress version as a measure to see if it helped the test run time. I can't find the issue to bump it back up... Can we check and see if this impacts the running time of some of the e2es?

Good point, completely forgot about that. Will remove it for now 👍

Comment (j-pendlebury) [reply package.json]: Will also create an issue for investigating the new version of Cypress

Comment (j-pendlebury) [reply package.json]: Issue for updating Cypress: https://github.com/bbc/simorgh/issues/5601

**Review by Bopchy — APPROVED** [review]
Looks good to me 👍

**Review by 12 — APPROVED** [review]

**Review by thekp — APPROVED** [review]

**Comment by PriyaKR** [comment]
LGTM.

**Review by amywalkerdev — APPROVED** [review]

**Pull request merged**